utility_analysis_new/dp_engine.py

```diff
@@ -21,6 +25,70 @@
 import utility_analysis_new.combiners as utility_analysis_combiners
 
 
+@dataclasses.dataclass
+class MultiParameterConfiguration:
+    """Specifies parameters for multi-parameter Utility Analysis.
+
+    All MultiParameterConfiguration attributes corresponds to attributes in
+    pipeline_dp.AggregateParams.
+
+    UtilityAnalysisEngine can perform utility analysis for multiple sets of
+    parameters simultaneously. API for this is the following:
+    1. Specify blue-print AggregateParams instance.
+    2. Set MultiParameterConfiguration attributes which should have different
```

Comment:
Maybe just "Set MultiParameterConfiguration attributes (see the example below). Note that each attribute is a sequence of parameter values for which the utility analysis will be run. All attributes that have non-None values must have the same sequence size."?
Otherwise, I am not sure what "which should have different values" means in this context

Reply:
Thanks! I like that explanation.